include/revng/ADT/GenericGraph.h

```diff
@@ -228,6 +228,7 @@ class ForwardNode
   }
 
   bool hasSuccessors() const { return Successors.size() != 0; }
+  size_t successorCount() const { return Successors.size(); }
```

Comment:
In these situations LLVM does `successor_count` (see [Function::arg_size](https://llvm.org/doxygen/classllvm_1_1Function.html#ac6baa801e4aea800984e760d5460662f)). However this violates our coding conventions.

What do you think?

Reply:
ACK